.rspec

```diff
@@ -0,0 +1,6 @@
+--color
+<<<<<<< HEAD
```

Comment:
This seems to be a merge conflict that isn't solved yet. Please have a look on that.